server/pkg/item/item.go

```diff
@@ -9,9 +10,10 @@ import (
 type VersionedItem = version.Value[Item]
 
 type Item struct {
-	id       ID
-	schemaID schema.ID
-	fields   []*Field
+	id        ID
+	schemaID  schema.ID
+	projectID project.ID
```

Comment:
```suggestion
	projectID ProjectID
```

Due to the danger of circular references, do not import a package if you will only use the ID of that package; import only the id package.

Define `type ProjectID = id.ProjectID` at `pkg/item/id.go`